Assets/Scripts/TestGizmo.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class TestGizmo : MonoBehaviour {
+	public void OnDrawGizmos()
+	{
+		Gizmos.color = Color.red;
+		Gizmos.DrawWireSphere (new Vector3 (1, 1, 1), 0.5f);
```

Comment:
Was met belichting in mijn level aan het spelen, had een gizmo aangemaakt dat met belichting interacteerd, qua detection etc. Maar heb er verder niks mee gedaan.